build.assets/charts/Dockerfile-distroless

```diff
@@ -0,0 +1,25 @@
+ARG BASE_IMAGE=gcr.io/distroless/cc-debian11
+
+FROM debian:11 AS staging
+RUN apt-get update 
+COPY fetch-debs ./
+RUN ./fetch-debs dumb-init libpam0g libaudit1 libcap-ng0
+
+FROM debian:11 AS teleport
+# NOTE that the TELEPORT_RELEASE_SUFFIX *must* include the leading dash if set
+ARG TELEPORT_RELEASE_SUFFIX
+ARG TELEPORT_VERSION
+ARG TARGETARCH
+ENV TELEPORT_DEB_FILE_NAME=teleport${TELEPORT_RELEASE_SUFFIX}_${TELEPORT_VERSION}_${TARGETARCH}.deb
+COPY $TELEPORT_DEB_FILE_NAME ./$TELEPORT_DEB_FILE_NAME
+RUN dpkg-deb -R $TELEPORT_DEB_FILE_NAME /opt/staging && \
+    mkdir -p /opt/staging/etc/teleport && \
+    mkdir -p /opt/staging/var/lib/dpkg/status.d/ && \
+    mv /opt/staging/DEBIAN/control /opt/staging/var/lib/dpkg/status.d/teleport && \
+    rm -rf /opt/staging/DEBIAN
+
+FROM $BASE_IMAGE
+COPY --from=teleport /opt/staging /
+COPY --from=staging /opt/staging/root /
+COPY --from=staging /opt/staging/status /var/lib/dpkg/status.d
```

Comment:
If you're not handling post install scripts/similar for debs you may want to consider downloading tarballs instead. IMO doing so would reduce confusion when down the road we use more advanced deb features and don't get the expected result here. This would also reduce potential issues around distro-specific changes (as we have the ability to publish different packages for each distro).

Reply:
Most of the things I need are in the tarfile, but using the debian package makes generating the provenance information a bit easier. If it's not a big deal right now, I'll make an issue to come back and change the source of the teleport install.